src/sentry/ratelimits/redis.py

```diff
@@ -107,8 +107,15 @@ def is_limited_with_value(
         # Reset Time = next time bucket's start time
         reset_time = _bucket_start_time(_time_bucket(request_time, window) + 1, window)
         try:
-            result = self.client.incr(redis_key)
-            self.client.expire(redis_key, expiration)
+            pipe = self.client.pipeline()
+            pipe.multi()
```

Comment:
`multi` is not supported in redis-py either https://github.com/redis/redis-py/blob/master/redis/cluster.py#L2275-L2277

I think it's fine to rely on the pipeline, which is supported https://github.com/redis/redis-py/blob/master/redis/cluster.py#L785

Reply:
TIL

Reply:
Thank you @wedamija. I've force-pushed and removed multi